handlers.ts

```diff
@@ -0,0 +1,61 @@
+import fs = require("fs");
+
+const filePath = "file.txt";
+
+
+async function getUpdateTime() {
```

Comment:
use arrow functions
read in google `js function vs arrow function`

Reply:
done